moc_db_helper_functions.py

```diff
@@ -0,0 +1,44 @@
+import mysql.connector
```

Comment:
I understand that for you writing SQL is easier than learning how to use SQLAlchemy, however this PR would have been about 1/3 of the size and significantly more readable if you were to use it. I strongly believe it's worth the investment in the long term to understanding how it works.

I'm happy to help in any way I can.

Reply:
Thank you.  

Personally, I have never found ORMs to be all that useful as I have always wanted to do something special that is nearly impossible to do with the previous ORMs.  Usually ORMs do the run of the mill stuff more easily, though if you need to do something more, it becomes hideous to override and increasingly unreadable. 

Just for you, I will explore SQLAlchemy.

Reply:
I believe we should try to keep things as simple as possible. Therefore, if we're using ORM libraries in a way that makes things become hideous for us, then we should perhaps reconsider what we're doing, rather than drop the ORM library. That has held true for my years writing and reviewing software, and I've seen the same sentiment from other developers.

Reply:
Ideally they do everything that you state they do.  However, in the companies and projects where I have work with ORMs they have been massive failures.

Reply:
I don't disagree with you that there are certain situations that a ORM will not help. I have found those to be the exception rather than the general case, and I don't believe this or the software that we've been building is or has been one of those exceptions.

Reply:
I have written an example of implementing this using SQLAlchemy https://gist.github.com/knikolla/604797eb2972f0f7c2692ded076a3be8

Reply:
How about we place using SQLAlchemy as an issue as I have the impression that writing some tests a better use of tomorrow.